assets/trix/stylesheets/content.scss

```diff
@@ -1,4 +1,9 @@
 .trix-content {
+  h1 {
+    font-size: 1.6em;
+    margin: 10px 0;
```

Comment:
I'd suggest tweaking this margin to be em-based too. After playing with it a bit, I'd suggest `margin: 0.5em 0`.